pkg/uevent/utils.go

```diff
@@ -41,6 +43,23 @@ func getRootBlockPath(devName string) string {
 	}
 }
 
+func ValidateMountInfo(device *sys.Device, directCSIDrive *directcsi.DirectCSIDrive) bool {
```

Comment:
we should also check if the drive is umounted. ie, if drive.Status.Mountpoint != "" and device.MountInfos[0].MountPoint == ""

Reply:
generally, we need to check if device.Mounpoint is not matching with drive.Status.Mountpoint